code/ship/shiphit.cpp

```diff
@@ -1048,7 +1048,8 @@ void ship_hit_sparks_no_rotate(object *ship_objp, vec3d *hitpos)
 
 	int n = ship_p->num_hits;
 	if (n >= MAX_SHIP_HITS)	{
-		n = rand() % MAX_SHIP_HITS;
+		// DISCUSSME: was the minimum really intended to be 0?
+		n = util::Random::next(MAX_SHIP_HITS);
```

Comment:
Since 0 is explicitly handled below I think this is fine.

Reply:
Ok, good. I was just on the lookout for potentially odd-looking code